torch/testing/_internal/fake_config_module.py

```diff
@@ -28,6 +28,7 @@
 e_aliased_bool: bool = Config(
     alias="torch.testing._internal.fake_config_module2.e_aliasing_bool"
 )
+e_func: Callable = None
```

Comment:
Surprised ruff ruf013 didn't immediately flag and correct this.

Reply:
fixed now!